linalg/ode.cpp

```diff
@@ -106,6 +111,20 @@ std::unique_ptr<ODESolver> ODESolver::SelectImplicit(int ode_solver_type)
    }
 }
 
+std::unique_ptr<SplitODESolver> SplitODESolver::Select(int ode_solver_type)
+{
+   using ode_ptr = std::unique_ptr<SplitODESolver>;
+   switch (ode_solver_type)
+   {
+      case 55: return ode_ptr(new IMEXExpImplEuler);
+      case 56: return ode_ptr(new IMEXRK2);
+      case 57: return ode_ptr(new IMEXRK2_3StageExplicit);
+      case 58: return ode_ptr(new IMEX_DIRK_RK3);
+      default: MFEM_ABORT("Unknown ODE solver type: " << ode_solver_type );
+   }
```

Comment:
```suggestion
   switch (ode_solver_type)
   {
      case 55: return std::make_unique<IMEXExpImplEuler>();
      case 56: return std::make_unique<IMEXRK2>();
      case 57: return std::make_unique<IMEXRK2_3StageExplicit>();
      case 58: return std::make_unique<IMEX_DIRK_RK3>();
      default: MFEM_ABORT("Unknown ODE solver type: " << ode_solver_type );
   }
```
I believe this is entirely a superficial change... so feel free to ignore this suggestion

Reply:
I will keep the original as it will use the same style as the other Select methods.